presto-hive/src/test/java/com/facebook/presto/hive/metastore/TestCachingHiveMetastore.java

```diff
@@ -148,9 +157,10 @@ public void testGetTable()
         assertEquals(mockClient.getAccessCount(), 2);
     }
 
+    @Test
     public void testInvalidDbGetTable()
     {
-        assertFalse(metastore.getTable(TEST_METASTORE_CONTEXT, BAD_DATABASE, TEST_TABLE).isPresent());
+        assertFalse(metastore.getTable(TEST_METASTORE_CONTEXT, BAD_DATABASE, RANDOM_TABLE).isPresent());
```

Comment:
I think we should be able to change it back to TEST_TABLE with the new approach

Reply:
Because of the above stated reason, TEST_DATABASE name check cannot be added in getTable method in MockHiveMetastoreClient. So this should continue to make use of RANDOM_TABLE constant.